client/landing/stepper/declarative-flow/internals/steps-repository/completing-purchase/index.tsx

```diff
@@ -14,6 +14,7 @@ const CompletingPurchase: Step = function CompletingPurchase( { navigation, flow
 	const { __ } = useI18n();
 	const { submit } = navigation;
 	const { setPendingAction, setProgressTitle, setProgress } = useDispatch( ONBOARD_STORE );
+	const { setIntentOnSite } = useDispatch( SITE_STORE );
```

Comment:
Small, but I think we can consolidate this with the line below?

`const { setIntentOnSite, saveSiteSettings } = useDispatch( SITE_STORE );`